### ISSUE TITLE
No User ID in dataLayer on new registration

### ISSUE THREAD

**Issue by jtourns** [duracelltomi/gtm4wp#253]
Upon successful user registration I can see the Registration event firing in the GTM preview, but can't see the User ID in the datalayer.

The user isn't automatically logged in upon registration, could this be why the user ID isn't being pushed?

![registrationconfirmation](https://user-images.githubusercontent.com/16519517/213408738-87a7b357-823b-4c84-8584-770f2e180533.png)

**Comment by duracelltomi** [comment]
Yes, user ID (visitorId in GTM4WP) is only populated if there is a logged in user on your site. This is what GA also expects when seeing a user ID value.
